source/Pages/compute_on_spider.rst

```diff
@@ -305,7 +318,408 @@ combination that is not available you will receive the following error message:
+   srun --partition=gpu_v100 -gpus v100:1 --time=12:00:00 --x11 --pty bash -i -l
+   singularity shell --nv nv-tf-22.07.sif
+
+where USERNAME is your username and the partition is a GPU partition, like ``gpu_v100`` or ``gpu_a100`` depending on your project. The ``singularity shell`` command is needed to start jupyter from the command inside the container. The tutorials were cloned during the building of the image. The container is read-only, and some of the examples will require to download and store some files. To have writing functionality available for the examples, build the image with ``--sandbox`` and run it with ``--writable``.
+
+Start the notebook with:
+
+.. code-block:: bash
+
+   cd /tmp/docs/site/en/tutorials/keras
+   jupyter notebook --ip=0.0.0.0
+
+The python output will return an address like ``http://127.0.0.1:8888/?token=abc123``. Opening this address in your browser will give you access to the notebook. Now you can run an example from the ``keras`` folder. 
+
+.. WARNING::
+   Some jupyter instances provide a link of that contains ``hostname:8888``. Replace ``hostname`` with ``localhost`` or ``127.0.0.1`` to properly fetch the notebook.
+
+The terminal will now have CUDA output, while the notebook contains all the python and graphical output. Again, if there is an output in the terminal running the notebook similar to:
+
+.. code-block:: bash
+
+   2022-07-29 11:53:24.017428: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1532] Created device /job:localhost/replica:0/task:0/device:GPU:0 with 30987 MB memory:  -> device: 0, name: Tesla V100-PCIE-32GB, pci bus id: 0000:00:06.0, compute capability: 7.0
+ 
+this means the GPU is being used for your computations. Now you can run the classification (fashion) notebook and compare with the output of the `repository <https://www.tensorflow.org/tutorials/keras/classification>`_ to see if you get similar results.
+
+.. _resources-singularity: 
+
+Resources on singularity and containers
+=======================================
+
+| https://docs.sylabs.io/guides/latest/user-guide/
+| https://hub.docker.com/r/nvidia/cuda
+| https://catalog.ngc.nvidia.com/
+| https://gpucomputing.shef.ac.uk/education/creating_gpu_singularity (slightly outdated)
+
+Advanced GPU querying
+=====================
+
+Some of the GPU nodes in spider have multiple GPUs installed. This opens up the avenue where multiple users use the same node simultaneously. Here are some more advanced commands to explore a few options.
+
+To get one GPU and leave the other GPU on the node will available to someone else, do:
+
+.. code-block:: bash
+
+   srun -p gpu_a100 --gpus=a100:1 --pty bash
+
+To run on 2 GPUs simultaneously and have no other users on the nodes do:
+
+.. code-block:: bash
+
+   srun -p gpu_a100 --nodes=1 --exclusive --gpus=a100:2 --pty bash
+
+By default, half the cores of the node (22) are used when you use 1 out of 2 GPUs. To use only a single CPU core while using GPU do:
```

Comment:
This is likely only true for a certain type of GPU, perhaps add a link to the node descriptions and how GPUs (by default) are coupled to CPU cores